systemtest/src/test/resources/oauth2/oauth-metrics.yaml

```diff
@@ -0,0 +1,73 @@
+kind: ConfigMap
```

Comment:
Any reason why we don't have this in examples? Or why we don't use it from examples?